templates/default/jobwatcher.cfg.erb

```diff
@@ -4,4 +4,4 @@ scheduler = <%= node['cfncluster']['cfn_scheduler'] %>
 stack_name = <%= node['cfncluster']['stack_name'] %>
 cfncluster_dir = <%= node['cfncluster']['base_dir'] %>
 proxy = <%= node['cfncluster']['cfn_proxy'] %>
-instance_types_data = <%= node['instance_types_data'].to_json %>
\ No newline at end of file
+instance_types_data = <%= node['cfncluster']['instance_types_data'].to_json %>
```

Comment:
don't we need to update the default as well?

Reply:
Right! Thanks

Reply:
Done